doc/classes/Tween.xml

```diff
@@ -144,6 +144,12 @@
 				[b]Note:[/b] As it results from accumulating frame deltas, the time returned after the [Tween] has finished animating will be slightly greater than the actual [Tween] duration.
 			</description>
 		</method>
+		<method name="has_tweeners">
+			<return type="bool" />
+			<description>
+				Returns [code]false[/code] if neither [method tween_property], [method tween_interval], [method tween_callback], nor [method tween_method] is called, or the tween is invalid.
```

Comment:
```suggestion
				Returns [code]false[/code] if neither [method tween_property], [method tween_interval], [method tween_callback], nor [method tween_method] has been called, or if the tween is invalid.
```
Made the edit for this too late

Reply:
The vector `tweeners` will be cleared in `Tween::clear()` with the `valid` flag.